cli/src/api/templates/js-binding.ts

```diff
@@ -22,6 +22,8 @@ export function createCjsBinding(
 /* auto-generated by NAPI-RS */
 
 const { readFileSync } = require('fs')
+const { createRequire } = require('node:module')
+require = createRequire(__filename)
 
 let nativeBinding = null
```

Comment:
```suggestion
require = createRequire(__filename)
let nativeBinding = null
```

delete some whitespace so i can get github to sign a commit 🥴